README.md

```diff
@@ -159,15 +159,15 @@ kubectl apply -f <output_dir>/amko/crds/
 helm list -n avi-system
 
 NAME          	NAMESPACE 	REVISION	UPDATED                             	    STATUS  	CHART    	APP VERSION
-amko-1598451370 avi-system	1       	2022-05-19 10:00:31.609195757 +0000 UTC	    deployed	amko-1.9.1	1.9.1
+amko-1598451370 avi-system	1       	2022-05-19 10:00:31.609195757 +0000 UTC	    deployed	amko-1.10.1	1.10.1
```

Comment:
Might want to change updated date here as well as.
As per this document https://confluence.eng.vmware.com/pages/viewpage.action?pageId=1624722802, AKO/AMKO 1.10.1 got GAed on 9 June 2023

Reply:
updated